dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -131,7 +131,7 @@
     <Import Project="..\ComboBox\ComboBox.vcxitems" Label="Shared" Condition="$(FeatureComboBoxEnabled) == 'true' Or $(FeatureComboBoxEnabled) == 'productOnly'" />
     <Import Project="..\Pivot\Pivot.vcxitems" Label="Shared" Condition="$(FeaturePivotEnabled) == 'true' Or $(FeaturePivotEnabled) == 'productOnly'" />
     <Import Project="..\ScrollBar\ScrollBar.vcxitems" Label="Shared" Condition="$(FeatureScrollBarEnabled) == 'true' Or $(FeatureScrollBarEnabled) == 'productOnly'" />
-    <Import Project="..\CalendarView\CalendarView.vcxitems" Label="Shared" Condition="$(FeatureScrollBarEnabled) == 'true' Or $(FeatureScrollBarEnabled) == 'productOnly'" />
+    <Import Project="..\CalendarView\CalendarView.vcxitems" Label="Shared" Condition="$(FeatureCalendarViewEnabled) == 'true' Or $(FeatureCalendarViewEnabled) == 'productOnly'" />
     <Import Project="..\SplitView\SplitView.vcxitems" Label="Shared" Condition="$(FeatureScrollBarEnabled) == 'true' Or $(FeatureScrollBarEnabled) == 'productOnly'" />
```

Comment:
FeatureScrollBarEnabled for the SplitView project doesn't look correct either. The SplitView projects are also not included in the inner loop solution yet so this be done in a follow-up PR. @StephenLPeters FYI.

Reply:
Perhaps it would be better to create a standalone PR for this (and the currently missing CalendarView projects in the innerloop solution) instead of waiting for this PR to be merged. @ranjeshj @StephenLPeters Would you agree?